client/my-sites/plugins/marketplace/marketplace-test/index.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useSelector } from 'react-redux';
+import page from 'page';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'calypso/components/card-heading';
+import { Button, Card } from '@automattic/components';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import SidebarNavigation from 'calypso/my-sites/sidebar-navigation';
+
+export const Container = styled.div`
+	margin: 25px;
+	padding: 25px;
+`;
+
+export default function MarketplaceTest() {
+	const selectedSite = useSelector( getSelectedSite );
+	const marketplacePages = [
+		{ name: 'Plugin Details Page', path: '/marketplace/product/details/wordpress-seo' },
+		{ name: 'Loading Page', path: '/marketplace/product/setup' },
+		{ name: 'Domains Page', path: '/marketplace/domain' },
+		{ name: 'Thank You Page', path: '/marketplace/thank-you' },
+	];
+
+	return (
+		<Container>
+			<SidebarNavigation />
+			<CardHeading tagName="h1" size={ 32 }>
+				Marketplace Test Page With Links For Various Pages
```

Comment:
Not localizing since this page is not supposed to be seen by customers but as a simple testing helper for developers and for anyone having a look at this in the internal release.

Reply:
If this is intended for developers, should we wrap it in an environment check or better: a feature flag? We definitely don't want to unintentionally let this slip to production.

Reply:
The route is already wrapped around a feature flag. When releasing to customers we will not move this out of the feature flag.

Reply:
> When releasing to customers we will not...

Famous last words, but works for me 😉

Reply:
Other than wrapping the route in a feature flag what is a better way to do it @tyxla ? Do you mean to add a separate feature flag in addition to what is added as shown below? 
https://github.com/Automattic/wp-calypso/blob/640d4223234ec2ba2eb270a8c314484ca239ffcb/client/my-sites/marketplace/index.js#L22-L40

Reply:
Good question, let me elaborate. 

Right now, this is concealed behind the `marketplace-yoast` feature flag. So when we decide to roll this out to production, we'll enable this flag in production. But it's easy to forget that `/marketplace/test` is also under that same feature flag, so we'll end up exposing this to production unintentionally.

To combat that, you could add a separate feature flag as you suggested. That would make the intentions much clearer and would mitigate the potential rollout of the test interface to production users.

Reply:
I see got it! :)

Reply:
Fixed by 029aecb4f13e1bfd5b4964c6afe990eee8952e6f